bolt-modules/boltlib/lib/puppet/functions/run_plan.rb

```diff
@@ -11,6 +11,9 @@
   # @param args A hash of arguments to the plan. Can also include additional options.
   # @option args [Boolean] _catch_errors Whether to catch raised errors.
   # @option args [String] _run_as User to run as using privilege escalation.
+  #   This option sets the run-as user for all targets that accept the option
+  #   whenever Bolt connects and executes on the target throughout the plan run,
+  #   including sub-plans and sub-sub-palns.
```

Comment:
```suggestion
  #   including sub-plans.
```

I'm not sure that sub-sub-plans is needed.

Reply:
I think it clarifies that `_run_as` is applied not to just functions run *directly* with `_run_as`, but also indirectly.

Reply:
Heh, should probably spell "plans" right though :)

Reply:
Isn't that already implied by saying "including sub-plans"? The plan being run with `run_plan` is run directly with `_run_as`, but the sub-plans pick it up indirectly.

Would it be clearer to say that using `_run_as` with `run_plan` will indirectly apply the `_run_as` to other functions, including the `run_plan` function?

Reply:
M'k, I took another try at it!